### PR TITLE
Contact form section

### PR DESCRIPTION
**Pull request by Geoelycom**
This is the contact form section of the mobile Portfolio website.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -161,6 +163,42 @@ <h3 class="lang"><a href="#">Skills</a><i class="fas fa-angle-right"></i></h3>
...
+						<li>
+								<input type="email" name="email" placeholder="Johndoe@mail.com" required>
+						</li>
+						<li>
+								<textarea name="message" placeholder="write your message here" maxlength="500"></textarea>
+						</li>
+						<li class="form-button">
+							<button type="submit">Get in touch</button>
+						</li>
+					</ul>
```
Comment (mricanho) [inline index.html]: - [x] This is not actually a UL element but a form element,[ check this link](https://www.w3schools.com/html/html_form_elements.asp) to implement the correct semantic

File: index.html [hunk 2]
```diff
@@ -161,6 +163,42 @@ <h3 class="lang"><a href="#">Skills</a><i class="fas fa-angle-right"></i></h3>
...
+				<fieldset>
+					<ul>
+						<li>
+								<input type="text" name="name" placeholder="Elyan" maxlength="30" required>
+						</li>
+						<li>
+								<input type="email" name="email" placeholder="Johndoe@mail.com" required>
+						</li>
+						<li>
+								<textarea name="message" placeholder="write your message here" maxlength="500"></textarea>
```
Comment (mricanho) [inline index.html:190]: - [ ] You missed the type "textarea" in this input, and the tag it's definitely not a "textarea" but an input, please fix it.

Comment (mricanho) [inline index.html]: - [x] All the inputs must have the required attribute, you miss it here. Please add it

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @Geoelycom @ArthurGC,

The first stone for this milestone it's done, but...

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

- [x] You need a description for your Pull Request, remember that this is mandatory and for future projects, it could invalidate the PR
- [x] You need to add the name of your project to your README file, and delete the section that you are not using, such as "Getting Started"
- [x] From now on you need to improve your commit messages, [check this link](https://stackoverflow.com/questions/3580013/should-i-use-past-or-present-tense-in-git-commit-messages) about how to redact a correct commit message

_Check the comments under the review._

## Optional suggestions

- [ ] [OPTIONAL] There is an overflow in your X-axis, the reason could be a non-responsive width in your CSS file, please fix it. [Check this live demo](https://rawcdn.githack.com/Geoelycom/M-portfolio/2ce8a026a49ce77835ba0b6c1d8ea0edc1837281/index.html) of your page and open it in a mobile phone

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

work harder


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Geoelycom) [reply index.html]: I did have the required attribute to this, but web hint keeps flagging it as an error that it's not allowed(Linters provided as a requirement for the project) the reason I removed it, as I couldn't found a workaround without removing it.

File: index.html (hunk 1 above)
Comment (Geoelycom) [reply index.html]: Fixed. thanks

File: index.html (hunk 2 above)
Comment (Geoelycom) [reply index.html]: Got it at last.

File: README.md [hunk 3]
```diff
@@ -11,8 +9,6 @@ This project is been done to test my knowledge on Flexbox and semantic tags used
 - HTML,CSS,FLEXBOX,CSSGRID & JAVASCRIPT
 - Git & Github
 
-## Getting Started
-
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
```
Comment (ElijahTobs) [inline README.md]: - [x] As the first reviewer requested, you should delete the redundant sections (`Prerequisites`, `Setup`, etc) in your README.
- [x] _[Optional]_ Also, the text `To get a local copy up and running follow these simple example steps.` belongs to the __`Getting Started`__ section and that header has been deleted.
  - If you want the text to remain, you should add the header and also outline the steps to follow to get a local copy up and running

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required ⚠️ 🚦
## Hi @Geoelycom and @ArthurGC 👋

You've done a good job making the required changes requested by the previous reviewer.
_However, there's one more thing you have to do before this project can be approved._

## Required Changes ♻️

_See the inline comments below_

## Optional Changes 👌
_Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better._

#### Check for changes marked as Optional in the inline comments below.


_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

#### "Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."

### Note that you still have 1 more review as written in the [Code Reviews Limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561#), however, if you feel this review is unfair, you can request a second opinion using this [form](https://airtable.com/shrQAqnBwek5a0O0s).

## Happy Coding :computer: 

_Reviewed by_  **Elijah O. Ayandokun**

File: README.md (hunk 3 above)
Comment (Geoelycom) [reply README.md]: thank you. changes made

**Review by DeliceLydia — APPROVED** [review]
# Status: Approved 🎉 🎉 

Hi @Geoelycom , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional
- Please add in your README how to get the project up and running locally in the `GETTING STARTED SECTION`

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Geoelycom** [comment]
I had to remove this because the last TSE ask me to remove all redudancy. Can you all just stick to one thing. Its driving me bonkers right now.specifically the very thing you are asking me now.

**Pull request merged**